utils/actionAssembler.py

```diff
@@ -170,20 +173,68 @@ def replace_icons(actions_json):
 
 def replace_dynamic_values(value, session_id):
     if type(value) is str:
-        if value.startswith('$dynamic:'):
-            value = evaluate_dynamic_values(value.split(':')[-1], session_id)
+        if value.startswith(functions.DYNAMIC_START_SEQUENCE):
+            value = evaluate_dynamic_values(value.split(':')[1:], session_id)
```

Comment:
~~value.split(':')[-1]~~
value .split(functions.DYNAMIC_SEPARATOR)

Reply:
> ~value.split(':')[1:]
> ~value.split(functions.DYNAMIC_SEPARATOR)[]

Use the dynamic start sequence defined in the module
> ~DYNAMIC_START_SEQUENCE = '$dynamic'

Reply:
> ~value.split(':')[1:]~ value.split(functions.DYNAMIC_SEPARATOR)[]

Fixed